src/python/foglamp/device/__main__.py

```diff
@@ -16,25 +18,24 @@
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
-
+_logger = logger.setup("SouthBound Device", level=20)
```

Comment:
Can we wait until the directory restructure for the southbound/northbound changes?

Reply:
Keeping Device